code-challenges.js

```diff
@@ -57,12 +120,32 @@ const givenValue1 = 42
 const givenValue2 = 10
 // Expected output: 8
 
+//Function
+let lastIndex = (myNumbers.lastIndexOf(givenValue1))
+//console.log(lastIndex)
+//output: 7
+
+//let lastIndex = (myNumbers.lastIndexOf(givenValue2))
+//console.log(lastIndex)
+//output: 8
+
 // --------------------5) Create the code that will sort all the numbers in an array from largest to smallest. Use the test variables provided below.
 
 // Pseudo code:
+//create a code that takes the numbers in an array (example, sanDiegoSummerTemperatures and sanDiegoWinterTemperatures) and reassigns them as the variable largeSmall. Using .sort(), the output will display the numbers from the array from largest to smallest. 
 
 const sanDiegoSummerTemperatures = [79, 80, 72, 73, 82, 77, 76]
 // Expected output: [82, 80, 79, 77, 76, 73, 72]
 
 const sanDiegoWinterTemperatures = [59, 68, 62, 59, 66, 67, 66]
 // Expected output: [68, 67, 66, 66, 62, 59, 59]
+
+
+//Function
+var largeSmall = sanDiegoSummerTemperatures
+//console.log(largeSmall.sort())
+//output:[82, 80, 79, 77, 76, 73, 72]
+
+var largeSmall = sanDiegoWinterTemperatures
+//console.log(largeSmall.sort())
+//output: [68, 67, 66, 66, 62, 59, 59]s
```

Comment:
This one is a bit tricky.  You have the right idea, you just need to tweak the sort method a bit.  Check out these links about sort: [](https://github.com/learn-academy-2022-hotel/Syllabus/blob/main/javascript/arrays.md)
[](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort)